body.tex

```diff
@@ -85,37 +85,41 @@ \subsection{The Tools Used}
 \end{figure}
 
 \paragraph{MagicDraw}
-is the Rubin Observatory Modeling tool. It is used for maintaining the project model and for requirement management. 
-The Verification Elements are created in MagicDraw using the requirements as stating point, 
-then they are synchronized to and from Jira using \textbf{Syndeia}. 
-This ensures that the proper traceability between requirements and verification elements is in place, 
+is the Rubin Observatory Modeling tool. It is used for maintaining the project model and for requirement management.
+%not sure project model is descriptive enough
+The Verification Elements are created in MagicDraw using the requirements as a starting point,
+and are synchronized to and from Jira using \textbf{Syndeia}.
+This ensures that the proper traceability between requirements and verification elements is in place,
 and that changes in one part of the system can be propagated through other components.
 
 \paragraph{Jira}
 is the Rubin Observatory issue tracking system.
-The Adaptavist Test Manager plugin (ATM) provides additional functionalities to Jira to manage test activities.
-Jira is hosting all test information, providing to the user an easy-to-use graphic interface.
+The Adaptavist Test Manager plugin (ATM) provides additional functionality to Jira to manage test activities.
+Jira hosts all test information, providing an easy-to-use graphical interface to the user.
 
 \paragraph{Syndeia}
 is a third party tool that integrates MagicDraw and Jira. First the Verification Elements created in MagicDraw,
-are synched in Jira. After the test activity is completed, the information will be synched back to MagicDraw.
+are synced in Jira. After the test activity is completed, the information will be synced back to MagicDraw.
 
 \paragraph{Docsteady}
-is the tool that permits generation of the test documents, extracting the information from Jira using REST API.
-It can be executed manually or automatically, generating the documents in \LaTeX~format.
-It can be scheduled on a continuous integration tool like for example Jenkins, making possible that 
+is the tool that permits generation of the test documents, extracting the information from Jira using the REST API.
+It can be executed manually or automatically, and generates the documents in \LaTeX~format.
+%automaticall - perhpas clarify wiht a travis or jenkins job or what do you mean by autmatically .. cause next you speak of scheduling ..
```

Comment:
I think I have rephrase this accordingly to your suggestion: 

```
It can be executed manually -- from command line -- or automatically -- in a continuous integration (CI) tool -- 
and generates the documents in \LaTeX~format.
Using a CI system like for example Jenkins, makes possible to generate verification and validation documents continuously, or on a regular bases.
The extracted \LaTeX~ code is managed in Git repositories, following the  DM document workflow.
```